Editor/TextProvider.cs

```diff
@@ -222,10 +222,39 @@ internal sealed class TextProvider
             { Strings.AnywherePageLaunchClientLabel, "Run Game"},
             { Strings.AnywherePageLaunchClientButton, "Launch Client"},
             
-            { Strings.InfoLinkDocumentationLink, "Documentation" },
-            { Strings.InfoLinkForumLink, "AWS GameTech Forum" },
-            { Strings.InfoLinkTroubleshootingLink, "Troubleshooting" },
-            { Strings.InfoLinkReportIssuesLink, "Report Issues" },
+            { Strings.ManagedEC2Title, "Managed EC2"},
+            { Strings.ManagedEC2Description, "Managed EC2 fleets use Amazon EC2 instances to host your game servers.\nAmazon GameLift manages the instances and removes the burden of hardware and software management from hosting your games."},
+            { Strings.ManagedEC2IntegrateTitle, "Integrate GameLift with your game client"},
+            { Strings.ManagedEC2IntegrateDescription, "Add the Amazon GameLift server SDK to your game server and add code to communicate with the GameLift hosting service. If you're working with the Amazon GameLift sample game, this integration is already done."},
+            { Strings.ManagedEC2IntegrateLink, "Setup Amazon GameLift on a backend service"},
+            { Strings.ManagedEC2ScenarioTitle, "Select Deployment Scenario"},
+            { Strings.ManagedEC2ScenarioSingleFleetLabel, "Single-region Fleet"},
+            { Strings.ManagedEC2ScenarioSingleFleetRadio, "Creates a game backend service with a single Amazon GameLift fleet."},
+            { Strings.ManagedEC2ScenarioSingleFleetLink, "Learn more"},
+            { Strings.ManagedEC2ScenarioSpotFleetLabel, "Spot Fleet"},
+            { Strings.ManagedEC2ScenarioSpotFleetRadio, "Forms matches by using Amazon GameLift queues and a custom matchmaker and configures three fleets."},
+            { Strings.ManagedEC2ScenarioSpotFleetLink, "Learn more"},
+            { Strings.ManagedEC2ScenarioFlexFleetLabel, "Flex Match"},
+            { Strings.ManagedEC2ScenarioFlexFleetRadio, "Uses FlexMatch, a managed matchmaking service, to match game players together."},
+            { Strings.ManagedEC2ScenarioFlexFleetLink, "Learn more"},
```

Comment:
Please rename this to `ManagedEC2ScenarioFlexMatchLabel`. So basically all occurrences of `FlexFleet` -> `FlexMatch` since that's the internal terminology used.

Reply:
renamed